.github/integration_tests/nonexistent_repo.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Integration test to ensure CI fails when a non-existent repository is added
+
+# set git credentials
+git config user.name dfinity
+git config user.email dfinity@dfinity.org
+
+gh auth login
+
+# create new branch
+git checkout -b integration-test-1
+
+# add a fake repository
+echo "fake-repository" >> open-repositories.txt
```

Comment:
I believe this will always fail because of order, doesn't matter does this repository exist or not.

Reply:
The order is actually a separate workflow - I am only checking the results of `Check New Repository` - not the `Sorting` workflow. Though I could add a check to make sure that one fails too.